topydo/lib/ChangeSet.py

```diff
@@ -158,15 +163,13 @@ def _trim(self):
         for changeset in index[backup_limit:]:
             self.delete(changeset[0], p_write=False)
 
-    def get_backup(self, p_todolist):
-        """
-        Retrieves a backup for p_todolist from backup file and sets todolist,
-        archive and label attributes to appropriate data from it.
-        """
-        change_hash = hash_todolist(p_todolist)
-
-        index = self._get_index()
-        self.timestamp = index[[change[1] for change in index].index(change_hash)][0]
+    def get_backup(self, p_todolist=None, p_timestamp=None):
```

Comment:
I think the content of the method looks OK. Since this method doesn't return anything, the `get_` prefix is a bit of a misnomer. Suggestion is to rename it to `read_backup`.